pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -226,6 +227,22 @@ func (s *ServiceImpl) processAppPlugin(plugin plugins.PluginDTO, c *models.ReqCo
 	return nil
 }
 
+func (s *ServiceImpl) hasAccessToInclude(c *models.ReqContext) func(pluginID string, include *plugins.Includes) bool {
+	hasAccess := ac.HasAccess(s.accessControl, c)
+	return func(pluginID string, include *plugins.Includes) bool {
+		useRBAC := s.features.IsEnabled(featuremgmt.FlagAccessControlOnCall) && include.IsRBACReady()
```

Comment:
question: should it also be checked if RBAC is enabled here? I think the case of RBAC being disabled is also covered by `hasAccess(...)` call, but that makes the log message misleading and the logic slightly more complex in my opinion.